src/development/stack.yml

```diff
@@ -99,7 +99,6 @@ services:
     deploy:
       labels:
         - traefik.enable=true
-        - traefik.http.routers.adminer.middlewares=redirectscheme
```

Comment:
I think those should have a `#DARGSTACK-REMOVE` instead of being removed, no?

Reply:
idk, is https also used in dev mode? if not it's not needed at all

Reply:
yup